app/views/hyku_addons/dates/_date_fields.html.erb

```diff
@@ -1,16 +1,16 @@
-
-<label class="control-label" for="#{template}_date_#{date_type}"> Date  <%= "#{date_type}" %></label>
-<p class="help-block"><%= t("simple_form.hints.defaults.date_#{date_type}") %></p>
-
-
-<%= select_year(f.object.model.send("date_#{date_type}").to_s.split('-')[0].to_i, {start_year: (Date.today.year + 6), end_year: (Date.today.year - 159), prompt: "Select year"}, {id: "#{template}_date_#{date_type}__date_#{date_type}_year", name: "#{template}[date_#{date_type}][][date_#{date_type}_year]", class: "form-control ubiquity-date-#{date_type}-year ubiquity-date-input"} ) %>
-
-<%= select_month(f.object.model.send("date_#{date_type}").to_s.split('-')[1].to_i, { use_two_digit_numbers: true,
-                                                             prompt: 'select month if available'}, {id: "#{template}_date_#{date_type}__date_#{date_type}_month", name: "#{template}[date_#{date_type}][][date_#{date_type}_month]", :class => "form-control ubiquity-date-#{date_type}-month ubiquity-date-input"}) %>
-
-<%= select_day(f.object.model.send("date_#{date_type}").to_s.split('-')[2].to_i, { use_two_digit_numbers: true,
-                                                        prompt: 'select day if available'}, {id: "#{template}_date_#{date_type}__date_#{date_type}_day", name: "#{template}[date_#{date_type}][][date_#{date_type}_day]", :class => "form-control ubiquity-date-#{date_type}-day ubiquity-date-input"}) %>
-
-
-
-
+<div class="form-group">
```

Comment:
I didn't actually change this file, just reindented it and added the horizontal form markup.